src/js/track.js

```diff
@@ -44,7 +44,29 @@ Track.prototype.getSelectedSong = function() {
 };
 
 Track.prototype.normalizeSearchQuery = function(query) {
-	var normalized = query;
+	var removeableTags = ["official music video", "lyric video",  "out now",  "cover art",  "official video", "hd", "hq", "lyrics", "radio edit", "radio mix"];
```

Comment:
A few more that I've encountered: vinyl, full length video, lyrics on screen, live, w/lyrics, exclusive, animated, remastered.

Might also be worth adding a regex to replace hyphens/dashes (-) with a space for when things are entered as "Ramones-Cretin Hop" it still finds a match.